src/index.jsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter, Route } from 'react-router-dom';
+import AnalyticsRouter from 'react-g-analytics';
 import Site from './components/Site/Site';
 
-// TODO: Re-integrate <GoogleAnalytics analyticsId="UA-46921629-2" />
-// Consider `react-g-analytics` package
+const Router = process.env.NODE_ENV === 'production' ? AnalyticsRouter : BrowserRouter;
```

Comment:
We could use `require` here inside a conditional and only import the browser required for produciton. DOn't know if with this approach `BrowserRouter` gets dead code eliminated, if it does I'm fine with it.

Reply:
I think it get DCE, will confirm later.

Reply:
One more thing, shouldn't we keep the _analyticsId_ in a dot file so we don't expose it here, though we already did either way I guess 😅

Reply:
GA keys are public any way or do you mean just for configuration purposes? We can put it into package.json file.

Reply:
Yeah its safe to expose 👍 moving it out to single source of truth from the template is another question, which i suspect shouldn't be tackled in this PR